### PR TITLE
Release v1.6.2.post2

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Release DRUM 1.6.2.post2 for 7.3 backporting
* backporting [RAPTOR-7113]

## Rationale

[RAPTOR-7113]: https://datarobot.atlassian.net/browse/RAPTOR-7113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by yakov-g** [comment]
jarvis test functional_training_model_templates

**Review by jmbannon — APPROVED** [review]

**Comment by yakov-g** [comment]
jarvis test functional_training_model_templates

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

File: tests/functional/test_custom_task_templates.py [hunk 1]
```diff
@@ -378,7 +378,7 @@ def test_custom_task_templates(
                 # Set the target type in the metadata file sent to DataRobot to the correct type.
                 metadata = yaml.safe_load(open(metadata_filename))
                 metadata["targetType"] = target_type
-                yaml.dump(metadata, open(metadata_filename, "w"))
+                yaml.dump(metadata, open(metadata_filename, "w"), default_flow_style=False)
```
Comment (klichukb) [inline tests/functional/test_custom_task_templates.py:381]: what's this change doing?

**Review by klichukb — APPROVED** [review]

File: tests/functional/test_custom_task_templates.py (hunk 1 above)
Comment (yakov-g) [reply tests/functional/test_custom_task_templates.py:381]: This is part of https://github.com/datarobot/datarobot-user-models/commit/2b2b111c2a9dd0c1a1e4c1fa858a563b0f69b803
introduced after 1.6.2 release.
Another part of this PR I backported manually before.

**Pull request merged**